introduction.md

```diff
@@ -7,7 +7,7 @@ I am janak
 I am janak
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?